frontend/packages/ceph-storage-plugin/integration-tests/tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,254 @@
+    const pattern = 'rook-ceph-operator-';
+    const pod = getPodData(pods.items, pattern);
+    const phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+    expect(getPodName(pod).includes(pattern)).toBe(true);
+  });
+
+  it('Tests if nooba-operator is running', async () => {
+    const pattern = 'noobaa-operator-';
+    const pod = getPodData(pods.items, pattern);
+    const phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+    expect(getPodName(pod).includes(pattern)).toBe(true);
+  });
+
+  it('Tests if nooba-core is running', async () => {
+    const pattern = 'noobaa-core-';
+    const pod = getPodData(pods.items, pattern);
+    const phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+    expect(getPodName(pod).includes(pattern)).toBe(true);
+  });
+
+  it('Test if rook-ceph-mon-a is running', async () => {
+    const pattern = 'rook-ceph-mon-a-';
+    const pod = getPodData(pods.items, pattern);
+    const phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+    expect(getPodName(pod).includes(pattern)).toBe(true);
+  });
+
+  it('Test if rook-ceph-mon-b is running', async () => {
+    const pattern = 'rook-ceph-mon-b-';
+    const pod = getPodData(pods.items, pattern);
+    const phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+    expect(getPodName(pod).includes(pattern)).toBe(true);
+  });
+
+  it('Test if rook-ceph-mon-c is running', async () => {
+    const pattern = 'rook-ceph-mon-c-';
+    const pod = getPodData(pods.items, pattern);
+    const phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+    expect(getPodName(pod).includes(pattern)).toBe(true);
+  });
+
+  it('Test if rook-ceph-mgr is running', async () => {
+    const pattern = 'rook-ceph-mgr';
+    const pod = getPodData(pods.items, pattern);
+    const phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+    expect(getPodName(pod).includes(pattern)).toBe(true);
+  });
+
+  it('Test if 7 csi-cephfsplugin are running', async () => {
+    const pattern = 'csi-cephfsplugin-';
+    const podList = getPodData(pods.items, pattern);
+    expect(podList.length).toBe(7);
+    podList.forEach((pod) => {
+      expect(getPodPhase(pod)).toBe('Running');
+    });
+  });
+
+  it('Test if 7 csi-rbdplugin are running', async () => {
+    const pattern = 'csi-rbdplugin-';
+    const podList = getPodData(pods.items, pattern);
+    expect(podList.length).toBe(7);
+    podList.forEach((pod) => {
+      expect(getPodPhase(pod)).toBe('Running');
+    });
+  });
+
+  it('Test if 2 rook-ceph-mds-ocs-storagecluster-cephfilesystem pods are running', async () => {
+    const pattern = 'rook-ceph-mds-ocs-storagecluster-cephfilesystem';
+    const podList = getPodData(pods.items, pattern);
+    expect(podList.length).toBe(2);
+    podList.forEach((pod) => {
+      expect(getPodPhase(pod)).toBe('Running');
+    });
+  });
+
+  it('Tests if storage classes are shown', async () => {
```

Comment:
Don't we need to check for NooBaa StorageClass?

Reply:
noobaa storage class is not present.

Reply:
I think I've seen it.

Reply:
I just verified it. My bad it's present.